database.rules.json

```diff
@@ -33,6 +33,7 @@
           "$sid": {
             "options": {
               "$oid": {
+                ".write": "newData.exists() && data.parent().parent().child('addingOptionsIsPublic').val() === true",
```

Comment:
nit: can this be `optionsArePublic`? Seems more inline with the convention of `<type>(Is|Are)Public`

Reply:
- [ ] TODO: @MaxwellBo Update the `database.rules.json` in the Firebase console after this change is merged